sections/semantics-sections.include

```diff
@@ -1392,8 +1392,9 @@
           onExit(node)
       </pre>
 
-      ...exemplifies how to traverse the node tree and when to trigger the <i>entered</i> and
-      <i>exited</i> events.
+      ...exemplifies how to recursively traverse the node tree and when to trigger the <i>enter</i>
+      and <i>exit</i> events. See <a href="#code-fragments">Code fragments</a> for a possible,
+      non-recursive JavaScript implementation.
     </div>
```

Comment:
s/exemplifies/shows a way to/

Change the link? See comment on the change to add a header